src/app/examples/alert-example/alert-example.component.ts

```diff
@@ -76,6 +80,38 @@ this.modalController.showAlert(config);`;
     this.modalController.showAlert(config, this.onAlertClosed.bind(this));
   }
 
+  showAlertWithDynamicValues() {
+    const updateInterval = 1000;
+    const currentInterval = () => (60 * 1000) / updateInterval;
+    const toRemainingSeconds = (t: number) => currentInterval() - t;
+
+    const remainingSeconds$ = this.toggleCountdownTimer$.pipe(
```

Comment:
Not very readable - consider creating more simple example...

Reply:
Made the example simpler. Still, a timer is nice for showing the dynamicness